docs/units/FAIR-generics/topics/1-why-fair.md

```diff
@@ -115,7 +202,7 @@ references:
   - title: Ten reasons to share your data
     author: Amanda S. Barnard
     url: https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data
-  - title: "FAIR Cookbook: [What are the FAIR Principles?"
+  - title: FAIR Cookbook: What are the FAIR Principles?
```

Comment:
I am not sure whether the ":" can cause issues. They did sometimes in the old version.

Reply:
Maybe @martijnkersloot knows (and can adjust if necessary)

Reply:
I verified they did indeed cause errors. I removed them (see commits below).